### PR TITLE
task6

### PR DESCRIPTION
**Pull request by OleksandrKukla**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -16,15 +18,18 @@ import '../style.css';
 import "bootstrap/scss/bootstrap.scss";
 
 export const Content = () => (
-    <Router.Switch>
-        <Router.Route exact path="/" component={Home}/>
-        <Router.Route path="/details/:movieID" component={Details}/>
-    </Router.Switch>
+    <React.Fragment>
+        <Router.Switch>
+            <Router.Route exact path="/" component={Home}/>
```
Comment (xxxMortxxx) [inline src/components/App.js]: Looks like there is no need for React.Fragment here

File: src/components/Search.js [hunk 2]
```diff
@@ -1,34 +1,116 @@
...
+        this.state = {
+            searchParams,
+            searchString: searchParams.get(SEARCH_PARAM_NAME) || '',
+            searchBy: searchParams.get(SEARCHBY_PARAM_NAME) || SEARCH_TITLE
+        };
+    }
+
+    handleChange = (event) => {
+        this.setState({
+            ...this.state,
```
Comment (xxxMortxxx) [inline src/components/Search.js]: setState function merges what you pass to it, into existing state object, so there is no need to manually destruct ...this.state here. setState will do it under the hood

Read this article for more details: https://reactjs.org/docs/react-component.html#setstate

File: src/components/Search.js [hunk 3]
```diff
@@ -1,34 +1,116 @@
...
+            searchString: event.target.value || '',
+        });
+    };
+
+    setSearchType = (type) => {
+        this.setState({
+            ...this.state,
```
Comment (xxxMortxxx) [inline src/components/Search.js]: setState function merges what you pass to it, into existing state object, so there is no need to manually destruct ...this.state here. setState will do it under the hood

Read this article for more details: https://reactjs.org/docs/react-component.html#setstate

File: src/components/Search.js [hunk 4]
```diff
@@ -1,34 +1,116 @@
...
                             <small className="text-light">SEARCH BY</small>
                         </label>
-                        <Button title="TITLE" isActive={true} size="small" addClasses="mr-1"/>
-                        <Button title="GENRE" size="small" type="secondary"/>
+                        <Button title="TITLE"
+                                isActive={this.state.searchBy === SEARCH_TITLE}
+                                size="small"
+                                addClasses="mr-1"
+                                type="secondary"
+                                onClick={() => _this.setSearchType(SEARCH_TITLE)}
```
Comment (xxxMortxxx) [inline src/components/Search.js]: There is no need for _this trick here. You can use .this syntax, as you did with this.submit

Comment (OleksandrKukla) [reply src/components/Search.js]: babel throws error if `this` context uses, looks like babel issue:

**Pull request merged**